src/asciidoc/docs/guidelines/developerTip.PersistingDataAcrossSessions.adoc

```diff
@@ -0,0 +1,50 @@
+= Developer Tip: Persisting data across sessions
+:page-slug: persisting-data-across-sessions
+:page-description: How to persist data that doesn't expire with the session.
+
+Data can be stored temporarily using window.sessionStorage. However, data stored in this way will expire when the session ends. If data needs to be persisted across sessions, `window.localStorage` may be used. Data stored in this way has no expiration date. It will be available until the browser's cache is cleared.
+
+== Using local storage
+There are two ways to use `window.localStorage`.
+
+* Property names may used directly:
```

Comment:
These bullets are another example of passive voice where active voice, with imperative phrasing, would be more effective. For the bullet text, you could try something like,

- Use the property names. For example:
- Use the `setItem` and `removeItem` methods. For example:

The Microsoft Style Guide is a really excellent reference for a lot of common practices in technical writing. For a great explanation of tense and voice, check out the "Verbs" page: https://docs.microsoft.com/en-us/style-guide/grammar/verbs

Reply:
Done